test/integration/connect/envoy/run-tests.sh

```diff
@@ -90,10 +90,10 @@ function init_workdir {
   cp consul-base-cfg/* workdir/${DC}/consul/
 
   # Add any overrides if there are any (no op if not)
-  find ${CASE_DIR} -name '*.hcl' -maxdepth 1 -type f -exec cp -f {} workdir/${DC}/consul \;
+  find ${CASE_DIR} -maxdepth 1 -name '*.hcl' -type f -exec cp -f {} workdir/${DC}/consul \;
```

Comment:
Fixed an unrelated warning I was seeing executing `find` with args in the wrong order.

Reply:
Thats bsd vs gnu coreutils for you.